playwright.config.ts

```diff
@@ -1,20 +1,22 @@
 /** @type {import('@playwright/test').PlaywrightTestConfig} */
-import { devices } from '@playwright/test';
+import { devices, PlaywrightTestConfig } from '@playwright/test';
```

Comment:
I just wanted the autocompletion so i made the file a ts file and brought in the definition.